api/utils/validators.js

```diff
@@ -1,7 +1,9 @@
 const branchRegex = /^[a-zA-Z0-9._-]+$/;
+const githubUsernameRegex = /^[^-][a-zA-Z-]+$/;
 const shaRegex = /^[a-f0-9]{40}$/;
 
 module.exports = {
   branchRegex,
   shaRegex,
+  githubUsernameRegex,
```

Comment:
Sounds good, I wasn't sure if we wanted this to follow our branch naming conventions or not. I do think that github will reject repos (and possibly branches?) that begin with a leading `-` so it may be good to add that to the branch regex as well.

Reply:
Oh yeah, I think that’s the case. Good idea.

Reply:
Oh yeah, didn’t mean to imply that one should be replaced.

Reply:
I mixed up the repo and username validations!